modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -104,40 +103,31 @@
 	alt_titles = list("Nanotrasen Representative", "Nanotrasen Diplomat", "Central Command Representative")
 
 /datum/job/orderly
-
+	alt_titles = list("Orderly", "Medical Guard") //goof requested that alt dept guards titles be kept to [department] guard
```

Comment:
```suggestion
	alt_titles = list("Orderly", "Medical Guard") // departmental guards alt-titles should be kept to [department] guard to avoid confusion
```